README.md

```diff
@@ -10,64 +10,140 @@ Some reasons to use **membrane.term**:
 - If you need to programmatically control, inspect, drive, or manipulate a terminal
 - If you want to learn more about how terminal emulators work.
 - If you'd like to embed a terminal somewhere. There's not really a guide for embedding, but if you file an issue, I can provide some tips.
+- If you'd like to capture terminal screenshots for documentation
+
+## Installation
+
+Membrane.term works on macOS and Linux operating systems.
+
+To invoke via `clj -M:membrane.term` add the following alias to your `deps.edn`:
+```Clojure
+{:aliases
+ {:membrane.term
+  {:replace-deps
+   {phronmophobic/membrane.term {:git/url "https://github.com/phronmophobic/membrane.term.git"
```

Comment:
should be com.phronemophobic/membrane.term. I plan on putting membrane.term on clojars under the same name